tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -358,6 +358,13 @@ def get_moc_frontends(argv: argparse.Namespace):
 
 
 def prepare_ir(argv: argparse.Namespace):
+    # TODO: remove this workaround once new TensorFlow frontend supports non-frozen formats: checkpoint, MetaGraph, and SavedModel
+    # Now it converts all TensorFlow formats to the frozen .pb format in case new TensorFlow frontend
+    is_tf, _, _, _, _ = deduce_legacy_frontend_by_namespace(argv)
+    if argv.use_new_frontend and is_tf:
+        from openvino.tools.mo.front.tf.loader import convert_to_pb
+        convert_to_pb(argv)
```

Comment:
I think it's better to return path_to_tmp_pb_file from this method and use it here:

```
                if os.path.exists(path_to_tmp_pb_file):
                    os.remove(path_to_tmp_pb_file)
```

Reply:
Done